include/TMU.hpp

```diff
@@ -147,12 +152,13 @@ class TMU {
         },
         {
             //send every 2 seconds
-            .Key = CO_KEY(0x1800, 5, CO_UNSIGNED16 | CO_OBJ_D__R_),
+            .Key = CO_KEY(0x1800, 4, CO_UNSIGNED16 | CO_OBJ_D__R_),
```

Comment:
```suggestion
            .Key = CO_KEY(0x1800, 5, CO_UNSIGNED16 | CO_OBJ_D__R_),
```

This should still be 5. Changing it to 4 overlaps with a reserved value that you are not using. This is mostly an issue with how CANOpen dictionaries are created...